tests/testthat/test-FunctionReporter-class.R

```diff
@@ -272,6 +272,18 @@ test_that(".parse_R6_expression correctly ignores right side of list extraction"
 })
 
 
+
+test_that("FunctionReporter$private$extract_edges doesn't fail because of Carrots class", {
```

Comment:
Can you make the test description more descriptive of what it's checking for? Something along the lines of: "FunctionReporter R6 edge extraction handles all-singleton dependency edges properly".

Reply:
I updated the description, but I was a little torn, because the issue is not just all-singleton dependency, though that is what the test is actually testing.  The issue would have arisen if all of the R6 methods had exactly two dependencies, as well.  Testing that case would require a new test package, though.

Reply:
Perhaps "FunctionReporter R6 edge extraction handles case where all methods have the same number of dependencies"?